run_test.sh

```diff
@@ -168,7 +168,11 @@ function run_integration_test {
   export FLASK_ENV=integration_test
   export DC_API_KEY=
   export LLM_API_KEY=
-  export ENV_PREFIX=Staging
+  if [[ $ENV_PREFIX == "" ]]; then
+    export ENV_PREFIX=Staging
+  else
+    echo "Using ENV_PREFIX=$ENV_PREFIX"
```

Comment:
nit: maybe can always echo this even when the script manually sets the ENV_PREFIX to staging

Reply:
Done.